docs/description/description.json

```diff
@@ -176,7 +176,7 @@
 }, {
   "patternId" : "B323",
   "title" : "By default, Python will create a secure, verified ssl context for use in such classes as HTTPSConnection",
-  "description" : "B323: unverified\\_context",
+  "description" : "B323: unverified_context",
```

Comment:
I assume that the fix removes the \\?

Reply:
Yes, it was markdown but in the descriptions we use plain-text